makefile

```diff
@@ -303,6 +313,9 @@ tests-e2e-setup: setup-cluster
 		-e ARMADA_APICONNECTION_FORCENOTLS=true \
 		armada-executor --config /e2e/setup/insecure-executor-config.yaml
 
+	# Create test queue if it doesn't already exist
```

Comment:
Wouldn't this fail the build if this queue exists?

Reply:
No. Because of the `|| true`.